schema/me/artwork_inquiries.js

```diff
@@ -0,0 +1,43 @@
+import gravity from '../../lib/loaders/gravity';
+import Artwork from '../artwork/index';
+import { pageable, getPagingParameters } from 'relay-cursor-paging';
+import {
+  connectionDefinitions,
+  connectionFromArraySlice,
+} from 'graphql-relay';
+import {
+  GraphQLObjectType,
+  GraphQLID,
+  GraphQLNonNull,
+} from 'graphql';
+
+export const ArtworkInquiryType = new GraphQLObjectType({
+  name: 'ArtworkInquiry',
+  fields: () => ({
+    id: {
+      type: GraphQLID,
+    },
+    artwork: {
+      type: new GraphQLNonNull(Artwork.type),
+      resolve: ({ inquireable }) => inquireable,
+    },
+  }),
+});
+
+export default {
+  type: connectionDefinitions({ nodeType: ArtworkInquiryType }).connectionType,
+  args: pageable({}),
+  description: 'A list of the current user’s inquiry requests',
+  resolve: (root, options, request, { rootValue: { accessToken } }) => {
+    if (!accessToken) return null;
+    const { limit: size, offset } = getPagingParameters(options);
+    const gravityArgs = { size, offset, inquireable_type: 'artwork', total_count: true };
```

Comment:
it shouldn't change, that would be weird. not sure how this would get stored for a subsequent request though. isn't metaphysics dumb about whether or not the current request is somehow subsequent to a previous one? we'd have to cache this number somewhere, on a per-user basis? and expire it at some point?